lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -502,7 +502,16 @@ def create_enrollment_questionnaire_response(record: REDCapRecord, patient_refer
         'travel_states_phs_base',
         'swab_and_send_calc',
         'kiosk_calc',
-        'covid_test_week_base'
+        'covid_test_week_base',
+        'uw_housing_resident',
+        'uw_housing_group_a',
+        'uw_housing_group_b',
+        'uw_housing_group_c',
+        'uw_housing_group_d',
+        'uw_housing_group_e',
+        'uw_housing_group_f',
+        'uw_housing_group_g',
```

Comment:
Can we just consolidate these into one field `uw_housing_group` that then has a value of a-g?

Reply:
That does make things nicer. I checked with Jessica to see if we could have one field in REDCap, but the calculation logic doesn't seem easily doable, so we're keeping multiple group fields in REDCap, and I've combined them in the ETL. Thanks for that suggestion.